stacks/ecs.yml

```diff
@@ -401,13 +405,16 @@ Resources:
             03_fstrim_reclaim_disk:
               # http://docs.aws.amazon.com/AmazonECS/latest/developerguide/CannotCreateContainerError.html
               command: "(crontab -uroot -l 2> /dev/null; echo '*/5 * * * * docker ps -qa | xargs docker inspect --format=\"{{ .State.Pid }}\" | xargs -IZ /sbin/fstrim /proc/Z/root/') | crontab -u root -"
-            # 04_set_newrelic_data:
-            #   command: "echo \"display_name: $(curl -ss http://169.254.169.254/latest/meta-data/instance-id)\" >> /etc/newrelic-infra.yml"
-            # 05_install_newrelic_infra:
-            #   command: |
-            #     sudo curl -o /etc/yum.repos.d/newrelic-infra.repo https://download.newrelic.com/infrastructure_agent/linux/yum/el/6/x86_64/newrelic-infra.repo
-            #     sudo yum -q makecache -y --disablerepo='*' --enablerepo='newrelic-infra'
-            #     sudo yum install newrelic-infra -y
+            04_set_t2_unlimited:
+              command: !Sub |
```

Comment:
Hrmm... did i actually need a `!Sub` here?

Reply:
Seems like no

Reply:
it may actually bork because of the `${JSON}` if you keep the `!Sub`